suite-native/module-accounts-import/src/components/AccountImportOverviewCard.tsx

```diff
@@ -64,6 +64,7 @@ export const AccountImportOverviewCard = ({
                 </Box>
                 {shouldDisplayDeleteIcon && (
                     <IconButton
+                        data-testId="@account-import/coin-synced/delete-icon"
```

Comment:
```suggestion
                        data-testID="@account-import/coin-synced/delete-icon"
```

Reply:
It is strange that this is not picked by TS

Reply:
fixed